run.py

```diff
@@ -132,12 +132,17 @@ def main(args):
   parser.add_argument("-d", "--detectors",
                     nargs="*",
                     type=str,
-                    default=["null", "numenta", "random", "skyline",
-                             "bayesChangePt", "windowedGaussian", "expose",
-                             "relativeEntropy", "earthgeckoSkyline"],
+                    default=["null", "numenta", "random", "bayesChangePt",
+                             "windowedGaussian", "expose", "relativeEntropy",
+                             "earthgeckoSkyline"],
                     help="Comma separated list of detector(s) to use, e.g. "
                          "null,numenta")
 
+  parser.add_argument("--enableSkylineDetector",
```

Comment:
I don't think you need this extra argument. You can always just manually add 'skyline' to the command line argument for detectors even if it not in the default list.

Reply:
@subutai a valid point, changes made as requested - https://github.com/numenta/NAB/pull/335/commits/a4310ce086f873eda9a78eb76faa7a2dae4bab4f